app/kingdom/kingdom.py

```diff
@@ -0,0 +1,28 @@
+from datetime import date
+
+from app.people.knight import Knight
+from app.kingdom.contest import Contest
+
+
+class Kingdom:
+    def __init__(self, name: str, gold: int, armed_people: dict) -> None:
+        self.name = name
+        self.gold = gold
+        self.armed_people = armed_people
+        self.knights = dict()
+        print(f"The Kingdom of King {self.name} was established."
+              f" It has {len(armed_people)} people and {gold} gold.")
```

Comment:
The print statement uses a mix of single and double quotes. According to the checklist, you should use double quotes consistently throughout the code.